content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
> The various go metric abstractions are essentially wrappers, whether they claim so or not.

Those are utility functions, you're providing something extra rather than replacing core functionality of the client library.

> wouldn't it be better if we point them to a repo where they can join forces and even enjoy our consulting on best practices?

Some of this appears to be a little organisation specific.